app/recordtransfer/management/commands/schedule_jobs.py

```diff
@@ -23,12 +23,21 @@ def register_scheduled_jobs() -> None:
         settings.FILE_UPLOAD_ENABLED
         and settings.UPLOAD_SESSION_EXPIRE_AFTER_INACTIVE_MINUTES != -1
     ):
-        schedule = settings.UPLOAD_SESSION_EXPIRED_CLEANUP_SCHEDULE
+        cleanup_schedule = settings.UPLOAD_SESSION_EXPIRED_CLEANUP_SCHEDULE
         LOGGER.info(
             "Scheduling cleanup job (schedule: %s)",
-            schedule,
+            cleanup_schedule,
+        )
+        scheduler.cron(cleanup_schedule, func=cleanup_expired_sessions, queue_name="default")
+
+        email_schedule = settings.IN_PROGRESS_SUBMISSION_EXPIRING_EMAIL_SCHEDULE
+        LOGGER.info(
+            "Scheduling expiring email job (schedule: %s)",
```

Comment:
I think this is a bit clearer:

```suggestion
            "Scheduling periodic job to email reminders about expiring upload sessions (schedule: %s)",
```

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/409/commits/705310eb08ac8fc8cf0358e722024c5dd3bdea4f